nephos/helpers/misc.py

```diff
@@ -65,16 +64,15 @@ def execute_until_success(command, verbose=False, delay=15):
         res, err = execute(
             command,
             show_command=first_pass,
-            verbose=verbose and first_pass,
+            show_response=first_pass,
```

Comment:
This wont be needed if we remove the arguments from the execute as stated in the previous comment.